data.md

```diff
@@ -187,3 +201,8 @@ Operator `_++_` implements an append operator for sort `Stack`.
     rule #drop(TYPE VTYPES, < TYPE > VAL:Number : STACK) => #drop(VTYPES, STACK)
 endmodule
 ```
+
+Resources
+=========
+
+[WASM Spec]: <https://github.com/WebAssembly/design>
```

Comment:
The design repo is a bit scattered. Maybe use the spec repo instead, or link directly to the spec, fingers crossed they don't change the URL:
https://webassembly.github.io/spec/core/text/values.html#text-id

Reply:
Updated to use https://webassembly.github.io/spec

(so users can choose which version to view)